package.json

```diff
@@ -4,7 +4,8 @@
   "main": "index.js",
   "license": " MPL-2.0",
   "dependencies": {
-    "algob": ">=0.3.0"
+    "@algo-builder/algob": "^5.0.1",
+    "@algo-builder/web": "^5.0.1"
   },
   "scripts": {
     "algob": "algob"
```

Comment:
we should remove this, if you're using direct dependency. Otherwise add a node for `yarn link`

Reply:
Thanks. Changes done.